src/react/components/pages/editorPage/editorPage.test.tsx

```diff
@@ -100,7 +122,7 @@ describe("Editor Page Component", () => {
         const store = createStore(testProject, true);
         const props = MockFactory.editorPageProps(testProject.id);
 
-        videoPlayerPausedMock = Player as jest.Mocked<typeof Player>;
+        videoPlayerPausedMock = Player;
```

Comment:
why doesn't this need to be mocked anymore?

Reply:
Sonar was complaining about an unnecessary cast